forms-flow-web/src/containers/styles.scss

```diff
@@ -358,6 +358,9 @@ margin-right: 1rem;
     margin-left: 10px !important;
     display: inline-block;
 }
+.application-head-img{
+  margin-top: 5px;
+}
```

Comment:
can we add these margin-top as bootstrap class like mt-1 ?

Reply:
Yes, we can. Done the changes